src/static/stylesheets/defaults.scss

```diff
@@ -129,6 +129,22 @@ button {
   &.primary {
     background-color: $elv-color-mediumblue;
   }
+
+  &.radio-button {
```

Comment:
what is this radio button stuff about? I don't see it used in the diff

Reply:
A lot of the css changes are from fixing lint warnings that I somehow ignored when I made the save modal changes